wiki/Tournaments/Official_support/en.md

```diff
@@ -173,17 +170,19 @@ For all of these reasons (and more), we ask that profile badge prizes are only a
 
 ### What is considered "staff" as far as tournament management goes? {#what-is-considered-staff}
 
-We consider referees, mappool selectors, judges, team organizers, or any other form of managerial position directly involved in the maintenance and operation of a tournament to be a part of its staff.
+We consider referees, mappool selectors, playtesters, judges, team organizers, or any other form of managerial position directly involved in the maintenance and operation of a tournament to be a part of its staff.
```

Comment:
There doesn't look to be any agreed-upon name for mappool playtesters wiki-side, `mappool quality assurance team` reads too long and just `playtesters` could be kinda vague. I'll leave this up for discussion.
```suggestion
We consider referees, mappool selectors, mappool playtesters, judges, team organizers, or any other form of managerial position directly involved in the maintenance and operation of a tournament to be a part of its staff.
```

Reply:
Just playtester is fine. Anyone involved with tournaments knows what it refers to. What other than mappools would you be playtesting?